### PR TITLE
[Practices] More details in client profile

### PR DESCRIPTION
**Pull request by macfarlandian**
## Description of the change

Displays in the client profile all of the top-level client data we expect to have as of now (some more P1/P2 stuff might come in later but this is the basic set), including some additional fields that I added as part of this change based on what I have seen in https://github.com/Recidiviz/recidiviz-data/pull/11182. 

Screen Shot 2022-03-21 at 5 35 09 PM

## Type of change

- [x] New feature (non-breaking change that adds functionality)

## Related issues

> Closes https://github.com/Recidiviz/recidiviz-data/issues/11616

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/PracticesClientProfile/Details.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  letter-spacing: -0.01em;
+  line-height: 1.14;
+  margin-bottom: ${rem(spacing.sm)};
+  margin-top: ${rem(spacing.lg)};
+`;
+
+const DetailsList = styled.dl``;
+
+const DetailsSubheading = styled.dt`
+  color: rgba(53, 83, 98, 0.5);
```
Comment (Catacola) [inline src/core/PracticesClientProfile/Details.tsx:43]: Should we define this and the one on L52 in the palette?

**Review by Catacola — COMMENTED** [review]
This all looks good to me. I was able to get the site to run in offline mode, but I was unable to get the client profile to load. I tried `http://localhost:3000/practices/compliantReporting/100` and `http://localhost:3000/practices/compliantReporting`, but all I got were blank pages. Selecting an officer on `http://localhost:3000/practices` lead to nothing.

**Comment by macfarlandian** [comment]
yeah you have to hack in a value for `PracticesStore.selectedOfficers` for now until the selection UI is merged in (the existing select doesn't do anything it's just for show) ... may I suggest `["OFFICER1"]`

File: src/core/PracticesClientProfile/Details.tsx (hunk 1 above)
Comment (macfarlandian) [reply src/core/PracticesClientProfile/Details.tsx:43]: these aren't even named styles in Figma as yet so I am not sure what their status is re: the design system ... would be a question for @agronomic when he gets back maybe

**Comment by macfarlandian** [comment]
I was thinking about this actually and realized it could happen in the wild also, if a user is linked directly to this URL but doesn't have a default caseload. I will try to solve for that here!

**Comment by macfarlandian** [comment]
OK this page will now attempt to fetch the specified client's data if we don't already have it. @ohaibbq this decouples the client list from the subscription so it should also help somewhat with the list behavior by exposing fewer intermediate subscription states.

**Review by ohaibbq — APPROVED** [review]
🦮

**Pull request merged**